tasks/system_probe.py

```diff
@@ -444,19 +686,31 @@ def run_tidy(ctx, files, build_flags, fix=False, fail_on_issue=False, checks=Non
     ctx.run(f"clang-tidy {' '.join(flags)} {' '.join(files)} -- {' '.join(build_flags)}", warn=True)
 
 
-@task
-def object_files(ctx, parallel_build=True, kernel_release=None):
```

Comment:
Why remove this task ? It's pretty useful for us because sometime we only want to build object files but not the system probe binary

Reply:
What I realize is that our current workflow probably do no directly depends on rebuilding object files every time because until now it was very slow. What I would suggest is that we keep this task for now, until we fix all our non-obvious dependencies (I can think of at least 2 on CWS side), and then we can remove this function

Reply:
awesome thank you